### PR TITLE
Add Stackdriver monitoring agent

### PR DESCRIPTION
**Pull request by ocervell**
Add Stackdriver monitoring agent install to the startup script

### CODE REVIEW

**Comment by ocervell** [comment]
@aaron-lane done !

**Comment by kevensen** [comment]
@ocervell When you have a chance, could you please address the conflicts in CHANGELOG.md?

**Comment by ocervell** [comment]
@kevensen it should be all good, can we merge this ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -45,6 +45,10 @@ Version 4.0.0 is a backwards-incompatible release. Please see the [upgrade instr
 - Add on_gke submodule [#182]
 - Add `excluded_resources` variable to forseti_conf_server.yaml. [#213]
 
+### Added
```
Comment (gkowalski-google) [inline CHANGELOG.md]: (minor) Can you please move this change to the top of the changelog (line 9) and add it to the unreleased version. Looks good though, thanks for adding this!

`
## [Unreleased]

### Added

- Add Stackdriver monitoring agent to Forseti VMs [#177]

`

Comment (kevensen) [reply CHANGELOG.md]: @ocervell I apologize for the long delay.  Have you been able to review @gkowalski-google comments?  I'd be happy to get this merged into the next module release (5.1).

**Pull request closed without merging**